modules/images.py

```diff
@@ -156,10 +156,10 @@ def draw_texts(drawing, draw_x, draw_y, lines, initial_fnt, initial_fontsize):
             while drawing.multiline_textsize(line.text, font=fnt)[0] > line.allowed_width and fontsize > 0:
                 fontsize -= 1
                 fnt = get_font(fontsize)
-            drawing.multiline_text((draw_x, draw_y + line.size[1] / 2), line.text, font=fnt, fill=color_active if line.is_active else color_inactive, anchor="mm", align="center")
+            drawing.multiline_text((draw_x, draw_y + line.size[1] / 2), line.text, font=fnt, fill=ImageColor.getcolor(opts.grid_text_color_active, 'RGB') if line.is_active else ImageColor.getcolor(opts.grid_text_color_inactive, 'RGB'), anchor="mm", align="center")
 
             if not line.is_active:
-                drawing.line((draw_x - line.size[0] // 2, draw_y + line.size[1] // 2, draw_x + line.size[0] // 2, draw_y + line.size[1] // 2), fill=color_inactive, width=4)
+                drawing.line((draw_x - line.size[0] // 2, draw_y + line.size[1] // 2, draw_x + line.size[0] // 2, draw_y + line.size[1] // 2), fill=ImageColor.getcolor(opts.grid_text_color_inactive, 'RGB'), width=4)

```

Comment:
Maybe keep these lines as they are, but add

```
color_active = ImageColor.getcolor(opts.grid_text_color_active, 'RGB')
color_inactive = ImageColor.getcolor(opts.grid_text_color_inactive, 'RGB')
```
at the start of `def draw_grid_annotations()` so we do less work?

Ditto for `color_background` :)